File: 20200715/atm.js
function atm(num, times) {
  times.sort();
  let total = 0;
  let result = 0;

  for (let i = 0; i < num; i++) {
    total += times[i];
    result += total;
  }

  return result;
}

console.log(atm(5, [3, 1, 4, 3, 2]));

File: 20200715/atm.py
def atm(num, times):
    sortedTimes = sorted(times)
    total = 0
    result = 0

    for value in sortedTimes:
        total += value
        result += total

    return result


print(atm(5, [3, 1, 4, 3, 2]))

# n = int(input())
# arr = list(map(int, sys.stdin.readline().split()))
